cicd/gitops/configuring-sso-on-argo-cd-using-dex.adoc

```diff
@@ -10,7 +10,7 @@ After the {gitops-title} Operator is installed, Argo CD automatically creates a
 
 [IMPORTANT]
 ====
-The `spec.dex` parameter in the ArgoCD CR is deprecated. In a future release of {gitops-title} v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be removed. Consider using the `.spec.sso` parameter instead.
+The `spec.dex` parameter in the ArgoCD CR is deprecated. In a future release of {gitops-title} v1.10.0, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be removed. Consider using the `.spec.sso` parameter instead.
```

Comment:
We do not make such future promises in our docs, suggest leaving out the version number. Please check for guidelines/best practices for this one. Will not block the PR for this, though since the GA is done.